src/Core/Localization/CLDR/DataLayer/CurrencyCache.php

```diff
@@ -52,6 +52,13 @@ final class CurrencyCache extends AbstractDataLayer implements CurrencyDataLayer
      */
     protected $cache;
 
+    /**
+     * The lower data layer to communicate with (read/write).
+     *
+     * @var CurrencyDataLayerInterface|null
+     */
+    protected $lowerDataLayer;
```

Comment:
Must be moved in `AbstractDataLayer` and removed in LocaleCache & CurrencyCache & LocaleReference